app/models/motd_formatter_rss.rb

```diff
@@ -0,0 +1,16 @@
+require 'rss'
+
+class MotdFormatterRss
+
+  attr_reader :content
+
+  # @param [MotdFile] motd_file an MotdFile object that contains a URI path to a message of the day in OSC format
+  def initialize(motd_file)
+    feed = motd_file.content
+    @content = RSS::Parser.parse(feed)
```

Comment:
1. This can throw exceptions.

```
irb(main):009:0> RSS::Parser.parse "<Ddsak<dsalkl>"
RSS::NotWellFormedError: This is not well formed XML
malformed XML: missing tag start
Line: 1
Position: 14
Last 80 unconsumed characters:
<Ddsak<dsalkl>
        from /opt/rh/rh-ruby22/root/usr/share/ruby/rss/rexmlparser.rb:19:in `rescue in _parse'
        from /opt/rh/rh-ruby22/root/usr/share/ruby/rss/rexmlparser.rb:16:in `_parse'
        from /opt/rh/rh-ruby22/root/usr/share/ruby/rss/parser.rb:164:in `parse'
        from /opt/rh/rh-ruby22/root/usr/share/ruby/rss/parser.rb:79:in `parse'
        from (irb):9
        from /users/PZS0562/efranz/ondemand/dev/dashboard/vendor/bundle/ruby/gems/railties-4.2.7.1/lib/rails/commands/console.rb:110:in `start'
        from /users/PZS0562/efranz/ondemand/dev/dashboard/vendor/bundle/ruby/gems/railties-4.2.7.1/lib/rails/commands/console.rb:9:in `start'
        from /users/PZS0562/efranz/ondemand/dev/dashboard/vendor/bundle/ruby/gems/railties-4.2.7.1/lib/rails/commands/commands_tasks.rb:68:in `console'
        from /users/PZS0562/efranz/ondemand/dev/dashboard/vendor/bundle/ruby/gems/railties-4.2.7.1/lib/rails/commands/commands_tasks.rb:39:in `run_command!'
        from /users/PZS0562/efranz/ondemand/dev/dashboard/vendor/bundle/ruby/gems/railties-4.2.7.1/lib/rails/commands.rb:17:in `<top (required)>'
        from bin/rails:4:in `require'
        from bin/rails:4:in `<main>'
```

2. Also this assumes the RSS descriptions are formatted HTML. But in our case the format should be plain text that is markdown compatible:

https://github.com/OSC/rss_to_motd/blob/eed45d5338683dc276a6f02910a18cee6e50e59f/test/fixtures/motd_rss_example.xml

But I guess some sites might have just normal html descriptions... so maybe this is yet another formatter